src/main/java/org/spongepowered/api/asset/Asset.java

```diff
@@ -77,6 +77,33 @@ default void copyToFile(Path output) throws IOException {
     }
 
     /**
+     * Copies this Asset to the specified 'outputDirectory' {@Link Path}.
+     *
+     * @param outputDirectory The directory to copy to
+     * @throws IOException
+     */
+    default void copyToDirectory(Path outputDirectory) throws IOException {
+        checkNotNull(outputDirectory, "outputDirectory");
+        copyToFile(outputDirectory.resolve(this.getFileName()));
```

Comment:
Ok sure, I figured if plugins wanted to change the behaviour they could always extend copyToDirectory anyway.